lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -141,6 +149,7 @@ private:
 protected:
     uint32_t m_sessionId{ 0 };
     uwb::protocol::fira::UwbSessionStatus m_sessionStatus{};
+    std::vector<uwb::protocol::fira::UwbRangingData> m_rangingData{};
```

Comment:
During a ranging session, we should expect a *lot* of these. Apart from invoking the session callback with this data, is there any reason we need to save this?